packages/playground/website/cypress/e2e/app.cy.ts

```diff
@@ -104,7 +104,7 @@ describe('Query API', () => {
 			cy.wordPressDocument()
 				.find('[data-slug=gutenberg].active')
 				.should('exist');
-		});
+		}, 120_000);
```

Comment:
What's 120_000?

Reply:
@adamziel the _ is an ignored character when it appears directly between two digits. It's just a thousands separator.

This change was originally related to the e2e crash, which I suspected to be resource related, but I quickly determined that this wasn't the cause once I was able to get things locally, so I've reverted this change.